ot3_state_manager/ot3_state_manager/hardware.py

```diff
@@ -0,0 +1,104 @@
+"""File containing hardware dataclasses for OT3."""
+
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+from enum import Enum
+
+from ot3_state_manager.ot3_state_manager.measurable_states import Force, Position
+from ot3_state_manager.ot3_state_manager.ot3_axis import OT3Axis
+from ot3_state_manager.ot3_state_manager.pipette_model import PipetteModel
+
+
+class UpdatablePositions(Enum):
+    """Represents positions that can be changed on the OT3"""
+
+    GANTRY_X = "Gantry X"
```

Comment:
why do we need this instead of just using the axis?

Reply:
100b27c5fe94daf9508ee98fcb3e0ecc46012cec